{"name": "test_basic_addition", "status": "broken", "statusDetails": {"message": "pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Then \"the result should be 3\". Line 6 in scenario \"Basic addition\" in the feature \"/home/runner/work/bdd-test-poc/bdd-test-poc/features/example.feature\"", "trace": "E   pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Then \"the result should be 3\". Line 6 in scenario \"Basic addition\" in the feature \"/home/runner/work/bdd-test-poc/bdd-test-poc/features/example.feature\"\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "description": "/home/runner/work/bdd-test-poc/bdd-test-poc/features/example.feature: Basic addition", "start": 1749734124431, "stop": 1749734124432, "uuid": "b0721943-f73b-4d64-bdcb-f2c7bd2f7c7f", "historyId": "2d12d6d5fb792f388c7449c02e740ddc", "testCaseId": "2d12d6d5fb792f388c7449c02e740ddc", "fullName": "tests.test_example#test_basic_addition", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_example"}, {"name": "host", "value": "fv-az1380-605"}, {"name": "thread", "value": "1856-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_example"}]}